extensions/boot/src/main/java/org/vaadin/spring/boot/AtmosphereAutoConfiguration.java

```diff
@@ -20,7 +20,7 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.InitializingBean;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
-import org.springframework.boot.context.embedded.ServletListenerRegistrationBean;
+import org.springframework.boot.web.servlet.ServletListenerRegistrationBean;
```

Comment:
It is actually here as well: https://github.com/peholmst/vaadin4spring/pull/300/commits